fe/plugin-common/src/main/java/com/starrocks/types/Roaring64Map.java

```diff
@@ -923,6 +923,24 @@ public String toString() {
         return answer.toString();
     }
 
+    /**
+     * Display Bitmap using comma separated strings.
+     *
+     * @return A string separated by ",".
+     */
+    public String serializeToString() {
+        final StringBuilder answer = new StringBuilder();
```

Comment:
can we compress the csv in a quite simple way? 
1,2,3,4,7,9,100,101,102
=> use - to express ranges
1-4,7,9,100-102 # 1-4 means 1,2,3,4
=> introduce delta encoding: 
1:4,7,9,100:3 # 1:4 means there is a series of continuous integer whose length is 4 and the zeroth value is 1.

Reply:
Should keep compatibility with `StarRocks` core.